@stellar/lyra-api/webpack.config.js

```diff
@@ -10,6 +10,7 @@ const config = {
     index: path.resolve(__dirname, "./src/index.ts"),
   },
   output: {
+    globalObject: "this",
```

Comment:
this is needed for docusaurus to SSR lyra-api. Otherwise, it throws a `window` is not defined error